components/usage/pkg/db/team_membership_test.go

```diff
@@ -30,3 +31,70 @@ func TestTeamMembership_WriteRead(t *testing.T) {
 	require.NoError(t, tx.Error)
 	require.Equal(t, membership, read)
 }
+
+func TestListTeamMembershipsForUserIDs(t *testing.T) {
+	conn := db.ConnectForTests(t)
+	userWithTwoTeams := uuid.New()
+	memberships := []db.TeamMembership{
+		{
+			ID:     uuid.New(),
+			TeamID: uuid.New(),
+			UserID: uuid.New(),
+			Role:   db.TeamMembershipRole_Member,
+		},
+		{
+			ID:     uuid.New(),
+			TeamID: uuid.New(),
+			UserID: userWithTwoTeams,
+			Role:   db.TeamMembershipRole_Member,
+		},
+		{
+			ID:     uuid.New(),
+			TeamID: uuid.New(),
+			UserID: userWithTwoTeams,
+			Role:   db.TeamMembershipRole_Member,
+		},
```

Comment:
FYI, this looks good, but I'm not sure why this works, given that the specified type for `TeamMembership.SubscriptionID` is:

```
	SubscriptionID uuid.UUID          `gorm:"column:subscriptionId;type:char;size:36;" json:"subscriptionId"`
```

Does Go set this member to `nil` by default? Also FYI, I think we'll want to change this type specification, because `TeamMembership.SubscriptionID` can also be `""` (empty string -- its default value) [[1](https://github.com/gitpod-io/gitpod/blob/b1a36659fdccc6fa45771360a1672002d7c1312d/components/gitpod-db/src/typeorm/entity/db-team-membership.ts#L32-L37))

Reply:
Good spot. Will investigate what it stores the uninitialized values as. The nil value for a UUID is 0 as that's the nil value for a byte, it will depend on how the Value() interface for the DB is implemented. Will follow up on this.